vars/FreeBSD.yml

```diff
@@ -1,3 +1,11 @@
+__argus_radium_user: argus
+__argus_radium_user_home: /var/log/argus
+__argus_radium_user_shell: /usr/sbin/nologin
+__argus_radium_user_uid: 561
```

Comment:
sort of, yes. as long as the first role applied to a host is `ansible-role-employees`, no issues. but remember that this issue is a kind of corner case. most of packages create users. we do not have file servers, or other UID-sensitive hosts anymore. 

one possible issue i can think of is:

* run `ansible-role-employee`, creating a subset of employees
* run other roles that creates users with UID 1000~, such as this role
* later, other employees needs an account and you run `ansible-role-employees`
* the UID of the user has been used by the system user and the play fails